packages/nuxt/src/core/runtime/nitro/renderer.ts

```diff
@@ -157,6 +157,17 @@ async function getIslandContext (event: H3Event): Promise<NuxtIslandContext> {
   return ctx
 }
 
+const normalizeSsrError = (ssrErrorQuery: ReturnType<typeof getQuery>) => {
```

Comment:
The test is failing because the code added extra bundle size, do you want me:

- Reduce the amount of code
  - Remove the new added function and just parse statusCode in defineRenderHandler
- Or modify the threshold

Reply:
Let's try just parsing the statusCode in the render handler 🤞

If that doesn't work, then yes, we can bump the threshold...

Reply:
It seems that 90000 is matching the minimum size of the current bundle, I've moved statusCode casting inline and increase threshold slightly.

Reply:
sorry for the pain there - this looks great 👌